Extensions/FileSystem/filesystemtools.js

```diff
@@ -124,6 +124,23 @@ gdjs.fileSystem.makeDirectory = function (directory) {
   }
 }
 
+/**
+ * Save a string into a file.
+ * @param {string} text The text string to be saved
+ * @param {string} savePath The absolute path on the filesystem
+ */
+gdjs.fileSystem.saveTextToFile = function (text, savePath) {
+  const fileSystem = typeof require !== 'undefined' ? require('fs') : null;
+
+  if (fileSystem) {
+    fileSystem.writeFile(savePath, text, (err) => {
+      if (err) {
+        console.error("Unable to save the text to path: '" + savePath + "' " + err);
```

Comment:
Better use multiple arguments in this condition to have the full error object shown in the console, rather than just a shortened version of the error converted to text:

```
console.error("Unable to save the text to path: '" + savePath + "': ", err);
```